src/quo2/components/code/code/style.cljs

```diff
@@ -0,0 +1,84 @@
+(ns quo2.components.code.code.style
+  (:require [quo2.foundations.colors :as colors]))
+
+;; Example themes:
+;; https://github.com/react-syntax-highlighter/react-syntax-highlighter/tree/master/src/styles/hljs
+(defn theme
+  [theme-key]
+  (case theme-key
+    :hljs-comment (colors/theme-colors colors/neutral-40 colors/neutral-60)
+    :hljs-title   (colors/custom-color-by-theme :sky 50 60)
+    :hljs-keyword (colors/custom-color-by-theme :green 50 60)
+    :hljs-string  (colors/custom-color-by-theme :turquoise 50 60)
+    :hljs-literal (colors/custom-color-by-theme :turquoise 50 60)
+    :hljs-number  (colors/custom-color-by-theme :turquoise 50 60)
+    :line-number  colors/neutral-40
+    nil))
+
+(defn text-style
+  [class-names]
+  (let [text-color (->> class-names
+                        (map keyword)
+                        (some #(when-let [text-color (theme %)]
+                                 text-color)))]
```

Comment:
Just a suggestion
We're trying to move away from using anonymous function literal `#` and use `fn` for better readability.

Reply:
Oh wow, why are we doing that?

When multiple parameters are receivied I'd say it's better to use `fn`, but I wouldn't remove the entire use of  `#()`. :sweat_smile: 

I solved it for this PR.

Reply:
To be able to add a name for our parameters
It was usually because of [this](https://github.com/bbatsov/clojure-style-guide#no-multiple-forms-in-function-literals) but I think we I tried to tell you about it just in a generative way

Reply:
I already looked at the link you sent, and it doesn't apply to this segment of code since it mentions when we have multiple forms:
do:
```clojure
(fn [x]
  (form-1 x)
  (form-2 x)
  (form-3 x))
```
instead of:
```clojure
#(do
   (form-1 %)
   (form-2 %)
   (form-3 %))
```

But my code only had one form, `when-let`:

```clojure
#(when-let [var (a-fn %)]
   var) 
```

Also, it was receiving only one parameter, I think it was straightforward.

Anyway, thanks for providing the link :)